lerna-package-lock.json

```diff
@@ -2638,9 +2638,15 @@
 			"integrity": "sha512-Bt6ONbZJgKSK0WQcV6od9xqa6t4VcAKX6tRIwougvm/YJSc7YLz3itKR6g+ILaO0yH8+GnMHt6l11Bjsl44NBw=="
 		},
 		"@fluidframework/core-interfaces-previous": {
+<<<<<<< HEAD
```

Comment:
you have un-merged conflicts in this file

Reply:
Thank you!

Reply:
Thanks, I'm working on this.